substrate/utils/frame/benchmarking-cli/src/extrinsic/bench.rs

```diff
@@ -82,7 +82,7 @@ pub(crate) struct Benchmark<Block, BA, C> {
 }
 
 pub(crate) struct BenchmarkVer<Block, BA, C> {
-	client: Rc<RefCell<C>>,
```

Comment:
why exactly these changes are needed? im pretty sure it works just fine at the moment ? :thinking:

Reply:
Yes, it works just fine... But the rest of the relevant codebase uses Arc so I thought I'd switch to it entirely